packages/material-ui/src/TablePagination/TablePaginationActions.js

```diff
@@ -4,6 +4,8 @@ import KeyboardArrowLeft from '../internal/svg-icons/KeyboardArrowLeft';
 import KeyboardArrowRight from '../internal/svg-icons/KeyboardArrowRight';
 import useTheme from '../styles/useTheme';
 import IconButton from '../IconButton';
+import LastPageIcon from '@material-ui/icons/LastPage';
```

Comment:
seems build is failing at this line. How do I use LastPage icon from this file? https://github.com/mui-org/material-ui/blob/8ea2df8a18fb4a1f8a89e755e0d7f05a9b8eed73/packages/material-ui-lab/src/internal/svg-icons/LastPage.js

Reply:
I think that you need to copy icons from material-ui-icons and put inside material-ui/internal folder

Reply:
I would expect these icons to be already available in the lab folder, for the Pagination component.

Reply:
@oliviertassinari there were no icons called `FirstPage` and `LastPage`, so copied them to **material-ui/internal**

Reply:
@oliviertassinari why the package icons is not added as material ui dependency?

Reply:
1. to avoid a circular dependency
2. people might not want to use Material Design icons: https://npm-stat.com/charts.html?package=@material-ui/core,@material-ui/icons